### ISSUE TITLE
[Question] Does the CLI application support Cache for Video subcommand?

### ISSUE THREAD

**Issue by jamesonknutson** [qarmin/czkawka#821]
I couldn't find any documentation on this after searching other issues / reading FAQ, but after playing with the application a little bit I noticed that the GUI application finishes it's workloads significantly faster than the CLI version when searching for duplicate Video Files. I know that the GUI application has the ability to use a caching system to avoid having to redo hashes for Videos it already hashed previously, does the CLI application do the same? Or perhaps there is a CLI argument I am missing when executing the command?

Here is an example of what I am executing from CLI:

```
.\cli.exe video --file-to-save .\cli-results.txt --directories C:\Users\J\Videos
```

I do not have specific benchmarks on time taken / total entries scanned, but I can get those benchmarks for a comparison if needed.

Thanks!

**Comment by ShareBugreports** [comment]
Yes, it seems to have caching. It is [documented](https://github.com/qarmin/czkawka/blob/master/instructions/Instruction.md)

But it is implemented very [weird](https://github.com/qarmin/czkawka/issues/1122)

**Comment by qarmin** [comment]
Cache is supported in multiple modes, also with similar videos
